app/Http/Requests/Admin/BandMember/UpdateBandMemberRequest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace App\Http\Requests\Admin\BandMember;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class UpdateBandMemberRequest extends FormRequest {
+	private $bandMember;
+	/**
+	 * Determine if the user is authorized to make this request.
+	 *
+	 * @return bool
+	 */
+	public function authorize() {
+		$this->bandMember = $this->route('bandMember');
+		return $this->user()->can('update', $this->bandMember);
+	}
+
+	public function rules() {
+		return [
+			'name' => 'required',
+			'email' => 'required|email|unique:users,email,' . $this->bandMember->user->id,
+			'language' => 'required|in:en,nl',
+			'bandmember' => 'required|array'
```

Comment:
this needs fields validation (required fields need be required)

Reply:
Also, it needs save and save and submit features

Reply:
what does the save and submit mean??